src/Backend/Modules/Users/Console/ResetPasswordCommand.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Backend\Modules\Users\Console;
+
+use Backend\Modules\Users\Engine\Model as BackendUsersModel;
+use Backend\Core\Engine\User as BackendUser;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+
+class ResetPasswordCommand extends ContainerAwareCommand
```

Comment:
Do you need ContainerAwareCommand or does it also work with it's parent class Command since I don't see you using the container in this command